tests/test_jobqueue.py

```diff
@@ -169,6 +178,26 @@ def test_inUpdater(self):
         finally:
             u.stop()
 
+    def test_time_units(self):
```

Comment:
I cannot find a reliable way to test the `float`, one time it succeeds, one time it fails. Do you know a way in which I can reliably test floats? I know for certain that floats work, is not testing them an option, because if the test starts on the wrong millisecond and rounds the wrong way, the test will fail.